### PR TITLE
Adding ability to get model plugin capabilities and cache path maps

### PR DESCRIPTION
**Pull request by tomikazi**

### CODE REVIEW

File: pkg/pluginregistry/registry.go [hunk 1]
```diff
@@ -102,6 +112,24 @@ func (r *PluginRegistry) discoverPlugin(port uint) {
 	log.Infof("Configuration model plugin %s discovered on port %d", plugin.ID, port)
 }
 
+func getRoPathMap(resp *api.ModelInfoResponse) path.ReadOnlyPathMap {
+	pm := make(map[string]path.ReadOnlySubPathMap)
+	for _, pe := range resp.ModelInfo.ReadOnlyPath {
+		// TODO: Implement conversion
```
Comment (tomikazi) [inline pkg/pluginregistry/registry.go:118]: I thought you had code that did this. It's not clear to me what the logic here should be... flattening map/map to array was easy, but if you have the code to reconstitute the map/map, there's no need to redo it.

Comment (adibrastegarnia) [reply pkg/pluginregistry/registry.go:118]: No I don't have but I don't think I need it as well at the moment. I think I just need RW paths.

**Review by adibrastegarnia — APPROVED** [review]

**Comment by adibrastegarnia** [comment]
retest this please

**Comment by adibrastegarnia** [comment]
retest this please

**Pull request merged**